src/encoded/static/components/static-pages/components/StackedBlockVisual.js

```diff
@@ -643,22 +647,25 @@ export class StackedBlockGroupedRow extends React.PureComponent {
             );
         }
 
-        var h4TitleProps = {
-            'className' : "text-500 text-ellipsis-container",
-            'data-tip' : group && typeof group === 'string' && group.length > 20 ? group : null
-        };
-
-        var childBlocks = !isOpen ? this.childBlocksCollapsed() : null;
-
-        var maxBlocksInRow = Math.max.apply(Math.max, _.pluck(_.pluck((childBlocks && childBlocks.props && childBlocks.props.children) || [], 'props'), 'data-block-count'));
+        var childBlocks = !isOpen ? this.childBlocksCollapsed() : null,
+            maxBlocksInRow  = childBlocks && Math.max.apply(Math.max, _.pluck(_.pluck((childBlocks && childBlocks.props && childBlocks.props.children) || [], 'props'), 'data-block-count')),
+            blockHeightFull = blockHeight + blockVerticalSpacing,
+            rowHeight       = blockHeight + (blockVerticalSpacing * 2) + 1;
 
         return (
```

Comment:
The output of this render() func is slightly changed in response to styling/CSS edits -- e.g. another element that wraps column label so that label may be centered vertically using inside a flexbox.